.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+on:
+  - pull_request
+  - push
+
+name: build
+
+jobs:
+  tests:
+    name: PHP ${{ matrix.php }}-${{ matrix.os }}-${{ matrix.coverage }}
+
+    env:
+      extensions: curl, mbstring, openssl, pdo, pdo_sqlite
+      SYMFONY_DEPRECATIONS_HELPER: weak
+
+    runs-on: ${{ matrix.os }}
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        php: ["5.6", "7.0", "7.1", "7.2"]
+        coverage: [xdebug]
+        experimental: [false]
+        include:
```

Comment:
Why testing 7.3+ against pcov only? XDebug 3 is released and should work well with `XDEBUG_MODE=coverage` in environment vairables.

Reply:
done.